build_scripts/android/build.sh

```diff
@@ -4,10 +4,9 @@
 
 buildpath=$1
 sourcepath=$2
-stl=$3
 
 if [[ -z "${buildpath}" || -z "${sourcepath}" ]]; then
-    echo "Usage: $0 <build path> <source path> [c++|gnustl]"
+    echo "Usage: $0 <build path> <source path> [c++]"
```

Comment:
We can drop the c++ bit now

Reply:
Will update at some other point, since this auto-merged.